integration-cli/requirements.go

```diff
@@ -40,7 +40,7 @@ var (
 		func() bool {
 			// Set a timeout on the GET at 15s
 			var timeout = time.Duration(15 * time.Second)
-			var url = "https://hub.docker.com"
+			var url = "http://www.baidu.com"
```

Comment:
Ah, sorry. It's quite slow to connect to docker hub in China so I changed the url temporarily but forgot to change it back before pushing to github.